crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/dict.py

```diff
@@ -48,3 +48,11 @@
     C: 0.1 * (10.0 / 12),
     D: 0.1 * (10.0 / 12),
 }
+
+# Regression test for formatter panic with comment after parenthesized dict value
+# Originally found in https://github.com/bolucat/Firefox/blob/636a717ef025c16434997dc89e42351ef740ee6b/testing/marionette/client/marionette_driver/geckoinstance.py#L109
+a = {
```

Comment:
This fix seems unrelated to if/elif right?

Reply:
it's already merged: https://github.com/astral-sh/ruff/pull/5293/files#diff-eb676c05c3588ddfa4f15b679292a4418e3b3d1bc5e17ce2ad9e150ff128097aR53

Reply:
just confirmed, it's also not part of the merge commit